Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -38,6 +38,118 @@ public final class System {
+                         (name: "BRD",           code: "BRD",   decimals: 18, symbol: "brd")]),
+
+        (id: "EOS Token", name: "EOS Token", code: "eos", type: "erc20", blockchainID: "ethereum-mainnet",
+         address: "0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0", verified: true,
+         demoninations: [(name: "EOS_INTEGER",   code: "EOSI",  decimals:  0, symbol: "eosi"),
+                         (name: "EOS",           code: "EOS",   decimals: 18, symbol: "eos")]),
+
+        (id: "Ripple", name: "Ripple", code: "xrp", type: "native", blockchainID: "ripple-mainnet",
+         address: nil, verified: true,
+         demoninations: [(name: "drop", code: "drop", decimals: 0, symbol: "drop"),
+                         (name: "xrp",  code: "xrp",  decimals: 6, symbol: "xrp")]),
+
+        // Testnet
+        (id: "Bitcoin-Testnet", name: "Bitcoin", code: "btc", type: "native", blockchainID: "bitcoin-testnet",
+         address: nil, verified: true,
+         demoninations: [(name: "satoshi", code: "sat", decimals: 0, symbol: BlockChainDB.Model.lookupSymbol ("sat")),
+                         (name: "bitcoin", code: "btc", decimals: 8, symbol: BlockChainDB.Model.lookupSymbol ("btc"))]),
+
+        (id: "Bitcoin-Cash-Testnet", name: "Bitcoin Cash Test", code: "bch", type: "native", blockchainID: "bitcoin-cash-testnet",
+         address: nil, verified: true,
+         demoninations: [(name: "satoshi",           code: "sat", decimals: 0, symbol: BlockChainDB.Model.lookupSymbol ("sat")),
+                         (name: "bitcoin cash test", code: "bch", decimals: 8, symbol: BlockChainDB.Model.lookupSymbol ("bch"))]),
+
+        (id: "Ethereum-Testnet", name: "Ethereum", code: "eth", type: "native", blockchainID: "ethereum-ropsten",
+         address: nil, verified: true,
+         demoninations: [(name: "wei",   code: "wei",  decimals:  0, symbol: BlockChainDB.Model.lookupSymbol ("wei")),
+                         (name: "gwei",  code: "gwei", decimals:  9, symbol: BlockChainDB.Model.lookupSymbol ("gwei")),
+                         (name: "ether", code: "eth",  decimals: 18, symbol: BlockChainDB.Model.lookupSymbol ("eth"))]),
+
+        (id: "BRD Token Testnet", name: "BRD Token", code: "brd", type: "erc20", blockchainID: "ethereum-ropsten",
+         address: BlockChainDB.Model.addressBRDTestnet, verified: true,
+         demoninations: [(name: "BRD_INTEGER",   code: "BRDI",  decimals:  0, symbol: "brdi"),
+                         (name: "BRD",           code: "BRD",   decimals: 18, symbol: "brd")]),
+
+        (id: "Ripple", name: "Ripple", code: "xrp", type: "native", blockchainID: "ripple-testnet",
+         address: nil, verified: true,
+         demoninations: [(name: "drop", code: "drop", decimals: 0, symbol: "drop"),
+                         (name: "xrp",  code: "xrp",  decimals: 6, symbol: "xrp")]),
+    ]
+
+    ///
+    /// Address Scheme
+    ///
+
+    var supportedAddressSchemesMap: [String:[AddressScheme]] = [
```

Comment:
This should really be at the C level, but time is a factor.

Reply:
agreed.